src/esw/mcu_reset.py

```diff
@@ -59,12 +59,12 @@ def reset_board(self) -> None:
 
         rospy.logerr("Resetting the entire MCU Board.")
 
+        self.time_of_last_reset = time()
```

Comment:
can u verify if this code is right or not

Reply:
Looks right to me